.circleci/config.yml

```diff
@@ -102,15 +102,15 @@ jobs:
   lint_license_unit_test_coverage:
     working_directory: ~/repo
     docker:
-      - image: cimg/base:stable-20.04
+      - image: cimg/openjdk:17.0-browsers
```

Comment:
Confirmed that this leads to an improvement for the lint job,

Reply:
I tried using the java browser image for the integration and smoke tests previously but the cypress tests started to fail related to cy.visit(). Any idea why this would happen?
https://app.circleci.com/pipelines/github/dockstore/dockstore-ui2/10130/workflows/9c796a92-81bb-4904-892c-7a4e465344de/jobs/44816
https://cloud.cypress.io/projects/1ya4vj/runs/29321/errors